war/src/main/less/base/style.less

```diff
@@ -1659,4 +1659,15 @@ svg.icon-xlg {
   display: none;
 }
 
+.app-summary {
+  & > td:first-of-type {
+    img,
+    svg {
+      width: 48px !important;
+      height: 48px !important;
+      margin-right: 1rem;
+    }
+  }
+}
```

Comment:
is this actually needed? I updated the icon class to use xlg in https://github.com/jenkinsci/jenkins/pull/6307/commits/35510e0e45f53fb0e7dfb41018339007e275e60a which fixed the issue afaik

Reply:
From what I saw it was causing a misalignment issue on pipeline jobs.